src/v/coproc/tests/CMakeLists.txt

```diff
@@ -43,3 +53,12 @@ rp_test(
   LIBRARIES v::seastar_testing_main v::coproc_test_utils v::kafka
   LABELS coproc
   )
+
+rp_test(
+  UNIT_TEST
+  BINARY_NAME wasm_event_listener_tests
+  SOURCES wasm_event_listener_tests.cc
+  DEFINITIONS BOOST_TEST_DYN_LINK
+  LIBRARIES v::seastar_testing_main v::coproc_test_utils
```

Comment:
same here, i think you can just add the `.cc` to a target and be done. boost was *designed* to combine tests suites.